### ISSUE TITLE
Assignment 02 - John

### ISSUE THREAD

**Issue by kylehill** [TIY-DC-FEE-Jan-2015/assignments#9]

**Comment by jodonne91** [comment]
https://github.com/jodonne91/html-simple-layout

couldn't figure out how to get the "login" positioned correctly. Also I know that most of these items are links but I was just worried about the positioning for now. I will update them as we continue.

**Comment by kylehill** [comment]
Looks good....

> "The Iron yard reported as the betst school ever"

Okay, looks good minus 3 points for spelling.

A few suggestions for next time:
1. You're right, all of those should be links. I'm sure you'll get more comfortable with positioning and styling and display modes as we continue this week, so things like those should hopefully be links during the Surf and Paddle assignment you'll get on Thursday.
2. The structure of your list's items is 

``` html
<span>Title</span>
<span>Domain<span>
<p>Score, author, and comments</p>
```

Like we covered today in class, I'd put the two spans inside of a block element to make sure they stick together.

3. We've covered it, but you've got that search bar as an unclosed `<input>` tag. All tags should be closed, or self-closing.
